### ISSUE TITLE
Provide a way to download docs offline

### ISSUE THREAD

**Issue by corysimmons** [w3c/tr-design#171]
Re: https://github.com/w3c/html/issues/573#issuecomment-243960648

**Comment by fantasai** [comment]
This is kinda out-of-scope for tr-design as well, we're just styling things here, not generating zip files. :) Happy to leave it open until someone finds a better home in the meantime.

**Comment by marcoscaceres** [comment]
I agree that it's a little bit outside the scope of TR-Design. What might be fun/useful would be adding a service worker when a document gets published on /TR/. I believe WHATWG specs do this already.
